setup.py

```diff
@@ -76,7 +76,7 @@ def run(self):
 
 
 install_requires = [
-    'nvidia-ml-py>=11.450.129,<=11.495.46',  # see #107
```

Comment:
Should we still pin the maximum `nvidia-ml-py` version? This patch only works for `nvmlDeviceGetMemoryInfo`. Upgrading the `pynvml` too aggressively may break `nvmlDeviceGet{Compute,Graphics}RunningProcesses_vN` when the v4 API is released with a new `nvmlProcessInfo_t` struct type.

Reply:
Hi @XuehaiPan, thanks for your comments. Actually I agree with you, because pynvml may break things without a notice and there is no way for downstream projects to prevent such unexpected changes perfectly beforehand. I do see that's similar to what you are doing in your nvitop project.